NAMESPACE

```diff
@@ -11,5 +13,7 @@ export(scale_fill_arcadia)
 export(show_arcadia_gradients)
 export(show_arcadia_palettes)
 export(theme_arcadia)
+import(ggplot2)
+import(snakecase)
```

Comment:
Generalized imports in the namespace should be avoided as much as possible. Are both of these necessary to achieve the functionality you were after @reillyowencooper?

Here's documentation for how to avoid these imports: https://r-pkgs.org/dependencies-mindset-background.html#sec-dependencies-namespace

Reply:
(because you're creating an s3, you might need these, but if it all possible it would be good to remove them)

Reply:
You're correct - I _think_ I need to keep the ggplot import because of the S3 extension, but definitely can take out the snakecase import. Bad habit from prior R package making. Will change

Reply:
Ok, tracked down what I needed to do - I don't need to import all of ggplot, but I do need to import the `ggplot_add` function. That function then makes `clean_plot` into the S3 object automatically. This means that both `S3method(ggplot_add,clean_plot)` and `importFrom(ggplot,ggplot_add)` go into `NAMESPACE` but I was able to remove `import(ggplot)`.

Reply:
ok perfect!